env.example

```diff
@@ -1,7 +1,12 @@
-COOKIE_PRIVATE_KEY=
-CONFIG_NAME=chis-ke
-PORT=3000             # for development environment
-EXTERNAL_PORT=3000    # for docker
-INTERFACE=0.0.0.0     # Leave as '0.0.0.0' for prod, suggest '127.0.0.1' for development
-CHT_DEV_URL_PORT=localhost:5984 # where your dev CHT instance is, hostname:port
-CHT_DEV_HTTP=true              # 'false' for http  'true' for https
+COOKIE_PRIVATE_KEY= 
+QUEUE_PRIVATE_KEY=                                    # should be different from COOKIE_PRIVATE_KEY
+CONFIG_NAME=chis-ke                                   # Name of the configuration
+PORT=3000                                             # for development environmentcontainer)
+EXTERNAL_PORT=3000                                    # for docker
+INTERFACE=0.0.0.0                                     # Leave as '0.0.0.0' for prod, suggest '127.0.0.1' for development
+REDIS_HOST=localhost                                  # Redis server hostname use 'redis' for docker
+REDIS_PORT=6378                                       # Redis server port
```

Comment:
suggest we leave these unset as both prod and dev will likely use the same  values.  As well, we should default to `redis` value (even though it's commented out):

```suggestion
#REDIS_HOST=redis                                  # Redis server hostname -  only uncomment if you know what you're doing
#REDIS_PORT=6378                                       # Redis server port -  only uncomment if you know what you're doing
```